app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -467,6 +465,28 @@ fun openLink(url: String, navController: NavController, useCustomTab: Boolean, u
     }
 }
 
+var prettyTime = PrettyTime(Locale.getDefault())
+var prettyTimeEnglish = PrettyTime(Locale.ENGLISH)
+val invalidPrettyDateRegex = "^[0123456789 ]+$".toRegex()
+fun formatDuration(date: Date, longTimeFormat: Boolean = false): String {
+    if (prettyTime.locale != Locale.getDefault()) {
+        prettyTime = PrettyTime(Locale.getDefault())
+    }
+
+    var prettyDate = prettyTime.formatDuration(date)
+
+    // A bug in PrettyTime means that some languages (pl, ru, uk, kk) will not include any time unit
+    if (prettyDate.matches(invalidPrettyDateRegex)) {
```

Comment:
Where's the bug in prettytime? It seems pretty active. Definitely open up a PR over there, and reference it here, so that we don't always have to use this hack.

Reply:
I linked the pretty time bug report in the PR description. 

But really, it's not so much a bug as a lack of support for these languages. For other languages, they have maps of words in those languages and logic to choose those words with the correct plurality and such.

However, for these four languages, the implementation does not contain any words or logic and just returns back whatever number was passed in.

I wrote this workaround here so that if/when the issue is fixed, it won't continue defaulting to English, it'll just become irrelevant.